packages/router/package.json

```diff
@@ -14,7 +14,7 @@
 	"dependencies": {
 		"@ionic/core": "^8.1.0",
 		"solid-element": "^1.8.0",
-		"@ionic-solidjs/core": "workspace:*"
+		"@ionic-solidjs/core": "1.0.1"
```

Comment:
I am not sure whether this will be a problem for bun (or other package managers?)